pontoon/sync/formats/silme.py

```diff
@@ -78,6 +85,9 @@ def __init__(self, parser, path, source_resource=None):
         self.source_resource = source_resource
         self.entities = OrderedDict()  # Preserve entity order.
 
+        # Bug 1193860: unescape quotes in some files
+        self.escape_quotes_on = 'mobile/android/base' in path and parser is DTDParser
```

Comment:
Is there any better heuristic for this? Is it bad if we escape all quotes in all DTD files?